.github/terraform/aws/main.tf

```diff
@@ -156,21 +204,20 @@ resource "aws_instance" "hazelcast_member" {
   }
 
   provisioner "file" {
-    source      = "~/lib/hazelcast.jar"
+    source      = var.hazelcast_path
     destination = "/home/${var.username}/jars/hazelcast.jar"
   }
 
   provisioner "file" {
-    source      = "hazelcast.yaml"
+    content     = data.template_file.hazelcast.rendered
     destination = "/home/${var.username}/hazelcast.yaml"
   }
 
   provisioner "remote-exec" {
     inline = [
       "cd /home/${var.username}",
       "chmod 0755 start_aws_hazelcast_member.sh",
-      "./start_aws_hazelcast_member.sh  ${var.aws_region} ${var.aws_tag_key} ${var.aws_tag_value} ",
-      "sleep 5",
+      "./start_aws_hazelcast_member.sh",
```

Comment:
When I was testing the tests became flaky when sleep was not there, not sure if anything has changed since then.

Reply:
There is already sleep in `start_aws_hazelcast_member.sh`. I'm wondering if we need both: one in script and one in `remote-exec`?